doc/packages/linux.section.md

```diff
@@ -2,40 +2,125 @@
 
 The Nix expressions to build the Linux kernel are in [`pkgs/os-specific/linux/kernel`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/kernel).
 
-The function that builds the kernel has an argument `kernelPatches` which should be a list of `{name, patch, extraConfig}` attribute sets, where `name` is the name of the patch (which is included in the kernel’s `meta.description` attribute), `patch` is the patch itself (possibly compressed), and `extraConfig` (optional) is a string specifying extra options to be concatenated to the kernel configuration file (`.config`).
+The function [`pkgs.buildLinux`](https://github.com/NixOS/nixpkgs/blob/f9d4f5f23ce7ccb7ad91d6d2150e367baf7014f6/pkgs/os-specific/linux/kernel/generic.nix) builds a kernel with [common configuration values](hthttps://github.com/NixOS/nixpkgs/blob/f9d4f5f23ce7ccb7ad91d6d2150e367baf7014f6/pkgs/os-specific/linux/kernel/common-config.nix).
+This is the preferred option unless you have a very specific use case.
+Most kernels packaged in Nixpkgs are built that way, and it will also generate kernels suitable for NixOS.
+[`pkgs.linuxManualConfig`](https://github.com/NixOS/nixpkgs/blob/f9d4f5f23ce7ccb7ad91d6d2150e367baf7014f6/pkgs/os-specific/linux/kernel/manual-config.nix) requires a complete configuration to be passed, and has less additional features than `pkgs.buildLinux`, such as the use of common configuration values and exposing the `features` attribute, as explained below.
```

Comment:
less is for uncountable *amounts*, fewer is for countable *numbers*.

It's not clear to me which is which in terms of features. Pick the one that's correct:


```suggestion
[`pkgs.linuxManualConfig`](https://github.com/NixOS/nixpkgs/blob/f9d4f5f23ce7ccb7ad91d6d2150e367baf7014f6/pkgs/os-specific/linux/kernel/manual-config.nix) requires a complete configuration to be passed.
It has fewer additional features than `pkgs.buildLinux`, which provides common configuration values and exposes the `features` attribute, as explained below.
```

```suggestion
[`pkgs.linuxManualConfig`](https://github.com/NixOS/nixpkgs/blob/f9d4f5f23ce7ccb7ad91d6d2150e367baf7014f6/pkgs/os-specific/linux/kernel/manual-config.nix) requires a complete configuration to be passed.
It has fewer additional features than `pkgs.buildLinux`, and for example uses common configuration values and exposes the `features` attribute, as explained below.
```

Reply:
I believe both would be correct, but I went with fewer in this case.